src/backend/revoker/subScan.ts

```diff
@@ -12,69 +12,141 @@ const SUBSCAN_MAX_ROWS = 100;
 const QUERY_INTERVAL_MS = 1000;
 const BLOCK_RANGE_SIZE = 100_000;
 
-const eventsApi = `https://${subscan.network}.api.subscan.io/api/scan/events`;
+const eventsListURL = `https://${subscan.network}.api.subscan.io/api/v2/scan/events`;
+const eventsParamsURL = `https://${subscan.network}.api.subscan.io/api/scan/event/params`;
```

Comment:
I also think it is very weird. I thought they wanted to get rid of all that didn't include `/v2/` on them, but no.

Reply:
I would rather have the full URL for the API at the beginning of the file than on the post-requests. I think it easies the overview of the api usage. But it is really just a matter of taste. 


Would this be more of your liking? 
 ```
const subscanAPI = `https://${subscan.network}.api.subscan.io/api`;
const eventsListURL = `${subscanAPI}/v2/scan/events`;
const eventsParamsURL = `${subscanAPI}/scan/event/params`;
```

Reply:
That’s exactly what I had in mind!

Reply:
Somehow this was not committed

Reply:
I rather used `const subscanAPI = 'https://${subscan.network}.api.subscan.io';`.

Why? Because the api-paths in their documentation always start with `/api/...` and they are made to copy fast. 

Check it out here: https://support.subscan.io/api-4669744